---
title: ActiveSyncException
second_title: Referencia de la API de Aspose.Email para .NET
description: Inicializa una nueva instancia delActiveSyncExceptionaspose.email/activesyncexception clase con un mensaje de error especificado.
type: docs
weight: 10
url: /es/net/aspose.email/activesyncexception/activesyncexception/
---
## ActiveSyncException(string) {#constructor_2}

Inicializa una nueva instancia del[`ActiveSyncException`](../../activesyncexception) clase con un mensaje de error especificado.

```csharp
public ActiveSyncException(string message)
```

| Parámetro | Escribe | Descripción |
| --- | --- | --- |
| message | String | Un mensaje que describe el error. |

### Ver también

* class [ActiveSyncException](../../activesyncexception)
* espacio de nombres [Aspose.Email](../../activesyncexception)
* asamblea [Aspose.Email](../../../)

---

## ActiveSyncException() {#constructor}

Inicializa una nueva instancia del[`ActiveSyncException`](../../activesyncexception) clase.

```csharp
public ActiveSyncException()
```

### Ver también

* class [ActiveSyncException](../../activesyncexception)
* espacio de nombres [Aspose.Email](../../activesyncexception)
* asamblea [Aspose.Email](../../../)

---

## ActiveSyncException(string, Exception) {#constructor_3}

Inicializa una nueva instancia del[`ActiveSyncException`](../../activesyncexception) clase con un mensaje de error especificado y una referencia a la excepción interna que es la causa de esta excepción.

```csharp
public ActiveSyncException(string message, Exception innerException)
```

| Parámetro | Escribe | Descripción |
| --- | --- | --- |
| message | String | El mensaje de error que explica el motivo de la excepción. |
| innerException | Exception | La excepción que es la causa de la excepción actual, o una referencia nula (Nada en Visual Basic) si no se especifica ninguna excepción interna. |

### Ver también

* class [ActiveSyncException](../../activesyncexception)
* espacio de nombres [Aspose.Email](../../activesyncexception)
* asamblea [Aspose.Email](../../../)

---

## ActiveSyncException(Exception) {#constructor_1}

Inicializa una nueva instancia del[`ActiveSyncException`](../../activesyncexception) clase con una referencia a la excepción interna que es la causa de esta excepción.

```csharp
public ActiveSyncException(Exception innerException)
```

| Parámetro | Escribe | Descripción |
| --- | --- | --- |
| innerException | Exception | La excepción que es la causa de la excepción actual, o una referencia nula (Nada en Visual Basic) si no se especifica ninguna excepción interna. |

### Ver también

* class [ActiveSyncException](../../activesyncexception)
* espacio de nombres [Aspose.Email](../../activesyncexception)
* asamblea [Aspose.Email](../../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.Email.dll -->
